src/Integrations/Goodbits/SettingsGoodbits.php

```diff
@@ -314,69 +232,54 @@ function ($option) use ($formId) {
-						<li>Copy the API key into the field below or use the global constant</li>
-					</ol>', 'eightshift-forms'),
-			],
-			[
-				'component' => 'divider',
-			],
+		$apiKey = Variables::getApiKeyGoodbits();
+
+		return [
+			$this->getIntroOutput(self::SETTINGS_TYPE_KEY),
 			[
-				'component' => 'checkboxes',
-				'checkboxesFieldLabel' => '',
-				'checkboxesName' => $this->getSettingsName(self::SETTINGS_GOODBITS_USE_KEY),
-				'checkboxesId' => $this->getSettingsName(self::SETTINGS_GOODBITS_USE_KEY),
-				'checkboxesIsRequired' => true,
-				'checkboxesContent' => [
+				'component' => 'tabs',
+				'tabsContent' => [
 					[
-						'component' => 'checkbox',
-						'checkboxLabel' => \__('Use Goodbits', 'eightshift-forms'),
-						'checkboxIsChecked' => $this->isCheckboxOptionChecked(self::SETTINGS_GOODBITS_USE_KEY, self::SETTINGS_GOODBITS_USE_KEY),
-						'checkboxValue' => self::SETTINGS_GOODBITS_USE_KEY,
-						'checkboxSingleSubmit' => true,
-					]
-				]
-			],
-		];
-
-		if ($isUsed) {
-			$apiKey = Variables::getApiKeyGoodbits();
-
-			$output = \array_merge(
-				$output,
-				[
+						'component' => 'tab',
+						'tabLabel' => \__('API', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'input',
+								'inputName' => $this->getSettingsName(self::SETTINGS_GOODBITS_API_KEY_KEY),
+								'inputId' => $this->getSettingsName(self::SETTINGS_GOODBITS_API_KEY_KEY),
+								'inputFieldLabel' => \__('API key', 'eightshift-forms'),
+								'inputFieldHelp' => \__('Can also be provided via a global variable.', 'eightshift-forms'),
+								'inputType' => 'password',
+								'inputIsRequired' => true,
+								'inputValue' => !empty($apiKey) ? 'xxxxxxxxxxxxxxxx' : $this->getOptionValue(self::SETTINGS_GOODBITS_API_KEY_KEY),
+								'inputIsDisabled' => !empty($apiKey),
+							],
+						],
+					],
+					$this->settingsFallback->getOutputGlobalFallback(SettingsGoodbits::SETTINGS_TYPE_KEY),
 					[
-						'component' => 'input',
-						'inputName' => $this->getSettingsName(self::SETTINGS_GOODBITS_API_KEY_KEY),
-						'inputId' => $this->getSettingsName(self::SETTINGS_GOODBITS_API_KEY_KEY),
-						'inputFieldLabel' => \__('API key', 'eightshift-forms'),
-						'inputFieldHelp' => \__('Can also be provided via a global variable.', 'eightshift-forms'),
-						'inputType' => 'password',
-						'inputIsRequired' => true,
-						'inputValue' => !empty($apiKey) ? 'xxxxxxxxxxxxxxxx' : $this->getOptionValue(self::SETTINGS_GOODBITS_API_KEY_KEY),
-						'inputIsDisabled' => !empty($apiKey),
+						'component' => 'tab',
+						'tabLabel' => \__('Help', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'steps',
+								'stepsTitle' => \__('How to get the API key?', 'eightshift-forms'),
+								'stepsContent' => [
+									\__('Log in to your Goodbits Account.', 'eightshift-forms'),
+									\__('Go to <strong>Settings</strong> page located in the sidabar.', 'eightshift-forms'),
```

Comment:
```suggestion
									\__('Go to <strong>Settings</strong> page located in the sidebar.', 'eightshift-forms'),
```
Sidabar: a bar you really wanna avoid 😂